auth.go

```diff
@@ -26,6 +33,11 @@ const (
 	authenticatorOAuth           = "OAUTH"
 	authenticatorSnowflake       = "SNOWFLAKE"
 	authenticatorOkta            = "OKTA"
+	authenticatorJWT             = "SNOWFLAKE_JWT"
+)
+
+const (
+	timeoutJWTSeconds = 60
```

Comment:
Can you make this a public variable so that the customers can set the validity? @howryu do you have any suggestion about the naming? I like `TimeoutJWTInSeconds`

Reply:
I would rename it to something like `jwtExpInSeconds`. However, in the server side, we had another check. No matter the `exp` value set in the jwt payload, it will always fail the authentication if `exp` is larger than 10 seconds, which is not ideal in case of client side need to retry the request.

I would defer exposing the parameter to customer.

Reply:
Sure. I saw the timeout is hardcoded as 60 second in JDBC so I chose this value. We can definitely exposed this as a settable parameter with default as 60.